aixplain/decorators/api_key_checker.py

```diff
@@ -0,0 +1,12 @@
+from aixplain.utils import config
+
+
+def check_api_key(method):
```

Comment:
Thanks for your contribution here! This is indeed valuable.

Reply:
Thank you for creating this Lucas. My question is if creating a decorator really the best option. It now needs to be imported and added on all the functions?
Just opening this up for discussion:
- we can have it maybe in the requests function when api key is header
- we cab maybe have on first import

Reply:
I removed the decorator from all the methods and now we are checking only when initializing the package, inside `__init__.py`.